Competition2018/src/main/java/competition/commandgroups/DynamicScoreOnScaleCommandGroup.java

```diff
@@ -35,11 +35,11 @@ public DynamicScoreOnScaleCommandGroup(
         this.addSequential(wait);
         // TODO: Uncomment these once the elevator/wrist is trustworthy.
         // Get ready to score
-        this.addParallel(setWristDown, 1);
-        this.addParallel(setElevatorForScale, 1);
+        //this.addParallel(setWristDown, 1);
```

Comment:
In retrospect it was obvious, but this program has the robot drive around pretty fast with the elevator all the way up. spooky.

I'm thinking to "fix" this by changing the PID to have the robot drive much slower, but hopefully still get to its goal. Alternatively we need ways to mark specific moments in the path where we want to do side tasks like this.

Reply:
An alternative would be to put the other command in a command group behind a plain delay command, all registered in parallel with PurePursuit — not perfect, but it would roughly do what you need.